rust_src/src/alloc.rs

```diff
@@ -112,4 +114,21 @@ pub fn purecopy(obj: LispObject) -> LispObject {
     }
 }
 
+/// Return a newly allocated marker which does not point to any place.
+#[lisp_fn]
+pub fn make_marker() -> LispMarkerRef {
+    //let val: LispObject = unsafe { allocate_misc(Lisp_Misc_Type::Lisp_Misc_Marker) };
+    //let mut p: LispMarkerRef = val.as_marker().unwrap();
+    let mut p = unsafe { allocate_misc(Lisp_Misc_Type::Lisp_Misc_Marker) }.force_marker();
+
+    // Set the properties of the marker to nothing
```

Comment:
The added comment is much appreciated, like with the variable names, the original C code can be quite obtuse sometimes.